src/Analysis/Engine/Impl/PythonAnalyzer.cs

```diff
@@ -206,7 +206,7 @@ public static PythonAnalyzer Create(IPythonInterpreterFactory factory, IPythonIn
         /// <returns>The project entry for the new module.</returns>
         public IPythonProjectEntry AddModule(string moduleName, string filePath, Uri documentUri = null, IAnalysisCookie cookie = null) {
             if (documentUri == null || documentUri.Scheme != "python") {
-                _pathResolver.AddModulePath(filePath);
+                // _pathResolver.AddModulePath(filePath);
```

Comment:
Do we need to comment out `SetRoot` as well?

Reply:
SetRoot above, you mean? My VS claims it has no references.

Reply:
The other option is that I can just comment out every reference to `_pathResolver` (declaration of the class field, construction, etc), but I thought that'd make merging for Alex more annoying and I didn't want to do that.

Reply:
`internal void SetRoot(string rootDir) => _pathResolver.SetRoot(rootDir);` called from

```
Analyzer.SetRoot(_rootDir);
```

in `DoInitializeAsync`

Reply:
Fixed.